source/Ship.cpp

```diff
@@ -2313,6 +2317,23 @@ void Ship::DoGeneration()
 	energy -= ionization;
 	fuel -= leakage;
 	heat += burning;
+
+	if(stackJumpDamage)
+	{
+		if(shields < stackJumpDamage)
+		{
+			stackJumpDamage -= shields > 0 ? shields : 0.;
+			shields = 0;
+			hull -= stackJumpDamage;
+			stackJumpDamage = 0;
+		}
+		else
+		{
+			shields -= stackJumpDamage;
+			stackJumpDamage = 0;
```

Comment:
Why bother resetting it?
It looks to me like with your current implementation it should not be stored, but maybe you intended to store the info within the ship
Honestly dont bother with it, it just happens when jumping

Reply:
I think this comment assumes the same as the comment below?
damage is applied once